cmd/kogito/command/install/kogito_operator_test.go

```diff
@@ -80,3 +82,44 @@ func TestInstallOperatorWithSupportServices(t *testing.T) {
 	assert.Contains(t, lines, "Jobs Service")
 	assert.Contains(t, lines, "Management Console")
 }
+
+func TestInstallOperatorWithDefaultChannel(t *testing.T) {
```

Comment:
follow the same pattern than the other tests, Test_Instal...

Reply:
done